libraries/EEPROM/examples/eeprom_write/eeprom_write.ino

```diff
@@ -0,0 +1,63 @@
+/*
+ * EEPROM Write
+ *
+ * Stores random values into the EEPROM.
+ * These values will stay in the EEPROM when the board is
+ * turned off and may be retrieved later by another sketch.
+ */
+
+#include "EEPROM.h"
+
+// the current address in the EEPROM (i.e. which byte
+// we're going to write to next)
+int addr = 0;
+#define N512 64
+void setup()
+{
```

Comment:
Please use "Format Sketch" feature in Arduino IDE when preparing example sketches.